parcels/particle.py

```diff
@@ -134,7 +134,8 @@ def __init__(self):
                 if time is None:
                     raise RuntimeError('Cannot initialise a Variable with a Field if no time provided. '
                                        'Add a "time=" to ParticleSet construction')
-                v.initial.fieldset.computeTimeChunk(time, 0)
+                if v.initial.grid.ti < 0:
+                    v.initial.fieldset.computeTimeChunk(time, 0)  # performance leak
```

Comment:
```suggestion
                    v.initial.fieldset.computeTimeChunk(time, 0)
```
Performance leak is fixed now, so not to confuse other developers I would remove the comment